core/services/workflows/engine.go

```diff
@@ -1263,7 +1263,7 @@ func NewEngine(ctx context.Context, cfg Config) (engine *Engine, err error) {
 	engine = &Engine{
```

Comment:
Note @cedric-cordenier @bolekk that this hex encoded value from `workflow.name = hex.EncodeToString([]byte(cfg.WorkflowName))` is passed into the engine, which has downstream consumers like

https://github.com/smartcontractkit/chainlink/blob/7dcc1fd68a080bdabffcaa54fd71e72657384194/core/services/workflows/engine.go#L442

Do capabilities expect a hex encoded string for workflow name? (maybe you expected long workflow names?)

Reply:
Yes, consumers expect hex encoded strings.